README.md

```diff
@@ -40,6 +40,7 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 
 <!-- Using https://github.com/yzhang-gh/vscode-markdown to manage toc -->
 - [Service Binding Specification for Kubernetes](#service-binding-specification-for-kubernetes)
+  - [- Example Resource](#--example-resource)
```

Comment:
This table of contents entry doesn't point anywhere.

Reply:
yea, the plugin I use to auto update the doc keep injecting this, I don't know why...

```suggestion
```